commit a158d5fa5aac4b451f3d3cc70e7d2f3e082b01d4 (refs/changes/26/326/1)
Author: Tim Gross <tim@0x74696d.com>
Date:   2016-08-22T10:20:12-04:00 (3 years, 2 months ago)
    
    Update README to point contributors to Gerrit
    
    For TOOLS-1530
    
    Per https://github.com/joyent/node-smartdc/pull/76#issuecomment-236633328
    the node-smartdc project has been moved to Gerrit. The README should point
    contributors to the Triton project contribution guidelines so we don't get
    PRs open on GitHub.

diff --git a/README.md b/README.md
index 2359168..aefe35b 100644
--- a/README.md
+++ b/README.md
@@ -3,6 +3,11 @@ the [Joyent SmartDataCenter API](http://apidocs.joyent.com/cloudapi/), for
 example the [Joyent Compute
 Service](http://www.joyent.com/products/compute-service).
 
+*This project does not use GitHub PRs.* See the [contribution guidelines]
+(https://github.com/joyent/triton/blob/master/CONTRIBUTING.md) and
+general documentation at the main [Triton project]
+(https://github.com/joyent/triton) page.
+
 (Note: Current releases and the #master branch of this are for SmartDataCenter
 (SDC) version 7.  It is not 100% backward compatible with SDC 6.5. For 100%
 compatility with SDC 6.5, you must install a "6.5.x" version of this module.)
@@ -43,7 +48,7 @@ the environment variable equivalents:
 
   Your matching SSH *private* key must be beside the ".pub" public key file
   in your "~/.ssh" dir.
-  
+
   If your client is connecting to a CloudAPI service that is using a self-signed
   SSL certificate, you may need to set `SDC_TESTING=1` in your client environent.
   (Otherwise you'll get `DEPTH_ZERO_SELF_SIGNED_CERT` error).
@@ -220,6 +225,11 @@ A few basic rules and guidelines:
   Be aware that this module is meant to work with older node versions
   and on a number of platforms (smartos, linux, mac, windows).
 
+- *This project does not use GitHub PRs.* See the [contribution guidelines]
+(https://github.com/joyent/triton/blob/master/CONTRIBUTING.md) and
+general documentation at the main [Triton project]
+(https://github.com/joyent/triton) page.
+
 
 ## Bugs
 
